webpack.dev.js

```diff
@@ -39,6 +40,7 @@ const DevConfig = merge.smartStrategy(
   },
   plugins: [
     new webpack.HotModuleReplacementPlugin(),
+    new ESLintPlugin()
```

Comment:
It may be just my environment, but anytime I run `HIDE_PDR=false SHOW_DISTRIBUTION_API_METRICS=true ENABLE_RECOVERY=true ESROOT=http://example.com APIROOT=http://localhost:5001 npm run serve` using `webpack.dev.js` I receive the error below.

```
Compiled with problems:

ERROR

../../.eslintrc.js#overrides[0] » eslint-config-airbnb-typescript/lib/shared:
	Configuration for rule "@typescript-eslint/camelcase" is invalid:
	Value {"properties":"never","ignoreDestructuring":false,"ignoreImports":false,"ignoreGlobals":false} should NOT have additional properties.
```
If it's just a local issue then I don't want to hold this PR up.

Reply:
Have not been able to repro this on my end. Also doesn't seem to be an issue when running on CI. Worth keeping note of though. It's particularly strange that it calls out typescript when we are not using typescript on the dashboard.

Reply:
That's a fair point. I don't want to hold up the PR because of some janky VSCode setting

Reply:
Especially when I was able to build, deploy and run all test prior to me deleting `node_modules` and `package-lock.json`